tools/dump_tool/ramdumpParser.py

```diff
@@ -618,6 +622,12 @@ def main():
 
 
 	try:
+		# Get arch family
+		if ('CONFIG_ARCH_FAMILY="armv8-m"' in data) or ('CONFIG_ARCH_FAMILY="armv7-m"' in data):
+			#If its cortex M, then run debugsymbolviewer script and return
```

Comment:
Pleaes change `its` to `it's`.
And this is only working with cortex M? Cannot with Cortex R?